### DISCUSSION TITLE
Q: 按照官方教程在Zeabur中部署MongoDB以及Waline项目，访问项目时出现500: Not initialized || Q: Follow the official tutorial to deploy MongoDB and Waline projects in Zeabur. When accessing the project, 500: Not initialized appears.

### DISCUSSION THREAD

**Discussion by nonhana** [walinejs/waline#2300] [Q&A] [answered]
### Describe the bug | 问题描述

按照官方Zeabur部署教程：[https://waline.js.org/guide/deploy/zeabur.html](https://waline.js.org/guide/deploy/zeabur.html)一步步进行部署，到最后生成自己的域名后进行访问。网站能够成功进入，但是评论无法发布、用户无法登录，出现500: Not initialized报错。

![刚进入报错](https://github.com/walinejs/waline/assets/105428118/3d1b131b-19ad-4a1a-8148-f502fbb47187)

![用户登录报错](https://github.com/walinejs/waline/assets/105428118/3781198e-9987-4687-88c9-db6f98ef3fec)

官方也并没有给出如何修复此问题的解决方案。

### Website URL | 问题网站

https://greyflowers-waline.zeabur.app/

### Where your waline deploy? | 服务部署在哪里？

Zeabur

### Where your comment data store? | 数据存储在哪里？

MongoDB

---
### Describe the bug | Problem description

Follow the official Zeabur deployment tutorial: [https://waline.js.org/guide/deploy/zeabur.html](https://waline.js.org/guide/deploy/zeabur.html) to deploy step by step, go to Finally, generate your own domain name and access it. The website can be accessed successfully, but comments cannot be posted, users cannot log in, and a 500: Not initialized error message appears.

![Error reported just after entering](https://github.com/walinejs/waline/assets/105428118/3d1b131b-19ad-4a1a-8148-f502fbb47187)

![User login error](https://github.com/walinejs/waline/assets/105428118/3781198e-9987-4687-88c9-db6f98ef3fec)

There is no official solution on how to fix this problem.

### Website URL | Problem website

https://greyflowers-waline.zeabur.app/

### Where your waline deploy? | Where is the service deployed?

Zeabur

### Where your comment data store? | Where is the data stored?

MongoDB

**Comment by lizheming** [comment, marked as answer]
这个是没有环境变量的问题，看下 MONGODB_URI 这个环境变量是否有，zeabur 应该会默认创建的。

---
This is a problem because there is no environment variable. Check whether the MONGODB_URI environment variable exists. Zeabur should create it by default.

**Reply by nonhana** [reply]
好的，感谢回复！已经成功修复问题。

---
OK, thanks for the reply! The problem has been successfully fixed.

**Comment by jongsh** [comment]
I am facing an issue where the MONGODB_URI environment variable is missing, even though MONGO_URI is present. What steps should I take, and where can I define the MONGODB_URI environment variable? @lizheming

**Reply by jongsh** [reply]
The part of environment variables of mongodb is like this:
```
MONGO_HOST=${CONTAINER_HOSTNAME}
MONGO_INITDB_ROOT_PASSWORD=${PASSWORD}
MONGO_INITDB_ROOT_USERNAME=mongo
MONGO_URI=${MONGO_CONNECTION_STRING}
MONGO_PASSWORD=${MONGO_INITDB_ROOT_PASSWORD}
MONGO_PORT=${DATABASE_PORT}
MONGO_USERNAME=${MONGO_INITDB_ROOT_USERNAME}
```
